lib/jsonapi/view.ex

```diff
@@ -94,23 +96,38 @@ defmodule JSONAPI.View do
         do: serialize(__MODULE__, models, conn)
 
       def url_for(nil, nil) do
-        "#{@namespace}/#{type()}"
+        case @host do
+        ""   -> "#{@namespace}/#{type()}"
+        host -> "#{host}#{@namespace}/#{type()}"
+        end
       end
 
       def url_for(data, nil) when is_list(data) do
-        "#{@namespace}/#{type()}"
+        case @host do
```

Comment:
These intentionally do not have a host because there is no connection, are we should we want to start appending the host to these?  It may have unintended consequences.